src/ragas/metrics/_context_recall.py

```diff
@@ -213,4 +216,78 @@ def save(self, cache_dir: str | None = None) -> None:
         self.context_recall_prompt.save(cache_dir)
 
 
+@dataclass
+class ContextRecall(LLMContextRecall):
+    name: str = "context_recall"
+
+    @deprecated(since="0.2", removal="0.3", alternative="LLMContextRecall")
```

Comment:
why is this depricated?

Reply:
`LLMContextRecall` is the new ContextRecall.